### ISSUE TITLE
Error entitlements - Application with SwG

### ISSUE THREAD

**Issue by matheus-viriato** [subscriptions-project/scenic-demo#8]
Hello good afternoon!

I am trying to implement an application with SwG for testing. I used your project and used it locally. I published an app on Google Play with billing support. I registered a product (productID) to do the integration with my web application.

I made the structured data on the publication page with the productID registered in the app and imported the SwG script into the page (https://news.google.com/swg/js/v1/swg.js).

Only I have the 404 error on the console with this GET:
https://news.google.com/swg/_/api/v1/publication/localhost/entitlements

In short, I've done all these steps:

- Create or select an existing project from the Google Developers Console. It is strongly recommended to use the project that is setup for Google Sign-in for the Subscribe with Google Publication API.

- Open the Subscribe with Google Public Library API and select Enable. The console will show information about the Subscribe with Google API in the Dashboard.

- Your project will need an OAuth 2.0 client ID. If you are using a project that already uses OAuth 2.0 you do not need to change your client ID. For information on creating a new client ID, see Setting up OAuth 2.0.

- The Subscribe with Google Publication API can only be accessed by whitelisted projects. To get API access for your project, please provide your project ID to your Subscribe with Google partner support point of contact.

Can you help me with this? Do I have to configure some swg API to make it run? The error is only giving in the entitlements API GET. Is there something left to set up?

I am waiting for answers and thank you for your attention!

**Comment by jpettitt** [comment]
Hi, We need to setup some things our side to enable your site.  Who did you talk to to get whitelisted?

**Comment by gerardolima** [comment]
hey, @jpettitt, is it still required to request for whitelisting?

**Comment by jpettitt** [comment]
Yes, a play account can't use Subscribe with Google without some additional config.  Are you working with anybody from Google? Please ping my email (same user name @google.com) and let me know who you are working with (or if you don't have a contact) and I'll get you connected with the right people.

**Comment by gerardolima** [comment]
hey, @jpettitt, thanks for your prompt response! by now, I'm doing some explorative research on this API for a future project and I hoped I could see all parts fitting together on a sample web application hosted in appspot; I undestand that's not the case, so I'll have to wait a little bit -- _please correct me if I'm wrong :)_

Again, and anyway, thank you very much for your response!
